app/src/main/java/com/sun/noteapp/ui/textnote/TextNoteActivity.kt

```diff
@@ -25,7 +33,51 @@ class TextNoteActivity : AppCompatActivity() {
         recyclerTextNoteLabel.apply {
             layoutManager = LinearLayoutManager(context, LinearLayoutManager.HORIZONTAL, false)
         }
+        imageButtonHeaderColorTextNote.setOnClickListener {
+            showImageColorDialog()
+        }
+        buttonAlarmTextNote.setOnClickListener {
```

Comment:
```suggestion
        buttonAlarmTextNote.setOnClickListener (this)
```
Với tất cả các view click